src/Store/FavoriteReducer.ts

```diff
@@ -0,0 +1,38 @@
+
+import {ImgPropsType} from "../Features/PhotosPage"
+import {useDispatch} from "react-redux";
+
+export const initialState: ImgPropsType[] =[];
+
+export type actionType = {type: string, payload: ImgPropsType}
+
+
+
+export const FavoriteReducer = (state = initialState, action: actionType): ImgPropsType[] => {
+
+ switch (action.type) {
+  case "addFavoriteImage": {
+      const isImgSame = state.filter (elem => elem.id === action.payload.id)
```

Comment:
`is` usually used for boolean vars naming, in this case better use `some` method of array to get bool value